adding mission, vision and values post (#4032)
unlock-protocol/unlock: unlock-protocol.com/blog/mission-vision.md

---
title: Mission, Vision and Values
subTitle: We empower creators to capture the value they create for their community, on their own terms
authorName: Julien Genestoux
publishDate: June 27, 2019
description: Identifying Mission, Vision and Values informs business strategy and paints a holistic and accurate picture of a business.
image: /static/images/blog/whatisunlock.jpg
---

A recipe to go from zero to one cannot exist. Creating a company, a team, a product or a protocol require a lot of work and faith. Our goal is ambitious, but it can only be achieved an incredible amount of small steps (4000 [Pull Requests](https://github.com/unlock-protocol/unlock/pulls) as of Tuesday!).

> This week, we took some time to think about our **mission**, which is what we do, our **vision**, which is why we do it and our **values**, which describe how we do it. We got some help and it took us a full day, but this work will help us in each decision and will act as a "scaffold" for the whole team.

Reviewer: ```suggestion
This week, we took some time to think about our **mission**, which is what we do, our **vision**, which is why we do it, and our **values**, which describe how we do it. We got some help and it took us a full day, but this work will help us in each decision and will act as a "scaffold" for the whole team.
```

## Our Vision

> We believe that, now, most creators are not able to capture enough of the value that they create. Open source developers, musicians, writters, or even curators provide an incredible value for all users of the internet and beyond, but we also have to acknoledge that most of them are not being rewarded for that work, and when they are, it is often through gatekeepers, middlemen who set the terms and too often end up competing with them.

Reviewer: ```suggestion
We believe that, now, most creators are not able to capture enough of the value that they create. Open source developers, musicians, writers, or even curators provide incredible value for all users of the internet and beyond. But we also have to acknowledge that most of them are not being rewarded for that work, and when they are, it is often through gatekeepers: middlemen who set the terms and too often end up competing with them.
```

> We know there is a better way and this is our vision:

Reviewer: ```suggestion
We know there is a better way. This is our vision:
```

> “Every creator can make a sustainable living from their work, while respecting the time, privacy and health of their community.”

## Our Mission

In order to make that vision a reality, we believe it starts with creators and this is why our mission is the following:

> "We empower creators to capture the value they create for their community, on their own terms."

## Our Values

Values define "how" we want to achieve our mission and realize the vision. They provide principles that we want to live by, both internally, but also externally, with users of the protocol and members of our own community. Here are the ones we chose:

- Collaborative
- Human Centered
- Mission Driven
- Simple
- Transparent

> Of course, in that list there are things that we already apply, and others which are, at least until now, more aspiratonial.

Reviewer: ```suggestion
Of course, in that list there are things that we already apply, and others which are, at least for now, more aspirational.
```

> Based on that list, it should be pretty clear that your input matters to us and we hope that you'll provide us guidance on how to adopt these principles during our journey! Please, reach out to us, in comments below, on [Telegram](https://t.me/unlockprotocol), or even privately using the Intercom widget in the lower right corner of this page.

Reviewer: ```suggestion
Based on that list, it should be pretty clear that your input matters to us. We hope that you'll provide us guidance on how to adopt these principles during our journey! Please, reach out to us, in comments below, on [Telegram](https://t.me/unlockprotocol), or even privately using the Intercom widget in the lower right corner of this page.
```
